here-naksha-lib-ext-manager/src/main/java/com/here/naksha/lib/extmanager/ExtensionCache.java

```diff
@@ -83,24 +82,25 @@ protected void buildExtensionCache(ExtensionConfig extensionConfig) {
       publishIntoCache(result, extensionConfig);
```

Comment:
Looks like we have reintroduced a gap in handing Extension "update" scenario, where previously loaded Extension needs to be shutdown first (using initClassName.close()).

Reply:
Added required logic to the code.